src/ets/scala/io/iohk/ethereum/ets/blockchain/BlockchainSuite.scala

```diff
@@ -0,0 +1,100 @@
+    "bcMultiChainTest/ChainAtoChainB_blockorder2" -> Set.empty,
+    "bcMultiChainTest/ChainAtoChainBtoChainA" -> Set.empty,
+    "bcMultiChainTest/UncleFromSideChain" -> Set.empty,
+    "bcTotalDifficultyTest/lotsOfBranches" -> Set.empty,
+    "bcTotalDifficultyTest/lotsOfBranchesOverrideAtTheMiddle"  -> Set.empty,
+    "bcTotalDifficultyTest/lotsOfLeafs"  -> Set.empty,
+    "bcTotalDifficultyTest/sideChainWithMoreTransactions" -> Set.empty,
+    "bcTotalDifficultyTest/uncleBlockAtBlock3afterBlock4" -> Set.empty,
+    "TransitionTests/bcFrontierToHomestead/blockChainFrontierWithLargerTDvsHomesteadBlockchain"  -> Set.empty,
+    "TransitionTests/bcFrontierToHomestead/blockChainFrontierWithLargerTDvsHomesteadBlockchain2"  -> Set.empty,
+    "TransitionTests/bcFrontierToHomestead/HomesteadOverrideFrontier" -> Set.empty
+  )
+
+  override def run(testName: Option[String], args: Args): Status = {
+    val options = TestOptions(args.configMap)
+    val scenarios = BlockchainScenarioLoader.load("ets/BlockchainTests/", options)
+
+    scenarios.foreach { group =>
+      group.name - {
+        for {
+          (name, scenario) <- group.scenarios
+          if options.isScenarioIncluded(name)
+        } {
+          name in new ScenarioSetup(scenario) {
+            if (unsupportedNetworks.contains(scenario.network)) {
+              cancel(s"Unsupported network: ${scenario.network}")
+            } else if (!supportedNetworks.contains(scenario.network)) {
+              fail(s"Unknown network: ${scenario.network}")
+            } else if (isCanceled(group.name, name)){
+              cancel(s"Test: $name in group: ${group.name} not yet supported")
+            } else {
+              log.info(s"Running test: ${group.name}/$name")
+              runScenario(scenario, this)
+            }
+          }
+        }
+      }
+    }
+
+    runTests(testName, args)
+  }
+
+  private def isCanceled(groupName: String, testName: String): Boolean =
+    ignoredTests.get(groupName).isDefined && (ignoredTests(groupName).contains(testName) || ignoredTests(groupName).isEmpty)
+
+  private def runScenario(scenario: BlockchainScenario, setup: ScenarioSetup): Unit = {
+    import setup._
+
+    loadGenesis()
+
+    val blocksToProcess = getBlocks(scenario.blocks)
+
+    val invalidBlocks = getBlocks(getInvalid)
+
+    val newBlocksErrors: Seq[BlockExecutionError] = processBlocks(blocksToProcess)
+
+    // If there is more block execution errors than expected invalidblocks, it means we rejected block which should
+    // pass validations, and the final state will not be correct
+    if(newBlocksErrors.size > invalidBlocks.size)
+      newBlocksErrors.foreach(err => log.info(err.toString))
+
+    newBlocksErrors.size shouldEqual invalidBlocks.size
+
+    val lastBlock = getBestBlock()
+
+    val expectedWorldStateHash = finalWorld.stateRootHash
+
+    lastBlock shouldBe defined
+
+    val expectedState = getExpectedState()
+    val resultState = getResultState()
```

Comment:
Account class does not have an address, maybe this should be a `List[Address -> Option[Account]]`?